### DISCUSSION TITLE
[QUESTION] Profile for fast print of cases/boxes with 1.0 Nozzle Volcano Direct Drive

### DISCUSSION THREAD

**Discussion by Kamik** [sn4k3/Ender3#39] [Q&A] [answered]
Hi, sn4k3, can you give advice/profile for fast print of boxes with 1.0/0.8 Volcano Nozzle on Direct Drive?

**Comment by sn4k3** [comment, marked as answer]
I don't have any set profile for 0.8mm/1.0mm + volcano, but it's easy to adapt.
If your model allows, you can run 1 perimeter with 1.2mm or 1.4mm extrusion walls, that alone will save a ton of time

```
0.8mm nozzle = 0.8mm > 1.6mm (Min - Max) Extrusions
1.0mm nozzle = 1.0mm > 2.0mm (Min - Max) Extrusions
```

For infills just set like 0.9mm extrusions or 1.1mm depending on the nozzle to save material.
For speeds just calculate the value based on: Layer height * extrusion width * speed (The result must be less than 30-25 mm/s^3) or it will skip the extruder due to much plastic to melt. But prusa slicer also have a global limitator for extrusion rate which you can set to 25 and max out the speed everywhere, it will slowdown when required...

For example, 0.8mm nozzle at 0.6mm layers height at a extrusion rate of 1.2mm, would set speed to: 35mm/s (0,6 * 1,2 * 35 = 25.2mm/s^3).

Volcano states that can run 35 to 40mm/s^3, but i never managed such rates! it start to skip between 25-30mm/s^3 and i dont want to boost temperature by alot
